code/datums/spells/chef.dm

```diff
@@ -0,0 +1,18 @@
+/datum/spell/chef/expert_chef/create_new_targeting()
+	return new /datum/spell_targeting/self
+
+/datum/spell/chef/expert_chef
+	name = "Expert Chef Knowledge"
+	desc = "Find things you can cook with the items in reach."
+	school = "chef"
+	panel = "Chef"
+	clothes_req = FALSE
+	base_cooldown = 5 SECONDS
+	human_req = TRUE
+	action_icon_state = "chef"
+	action_background_icon_state = "bg_default"
+	still_recharging_msg = "All this thinking makes your head hurt, wait a bit longer."
+
+/datum/spell/chef/expert_chef/cast(list/targets, mob/user = usr)
+	for(var/mob/living/carbon/human/H in targets)
```

Comment:
```suggestion
	var/mob/living/carbon/human/H = targets[1]
```
self targeting so we already know what's gonna be in here

Reply:
counterpoint: xenomicrobes

Reply:
Counterer point, requires a human to cast